coreax/kernel.py

```diff
@@ -335,9 +335,15 @@ def divergence_x_grad_y_elementwise(self, x: ArrayLike, y: ArrayLike) -> Array:
 
 
 class SquaredExponentialKernel(Kernel):
-    """
+    r"""
     Define a squared exponential kernel.
 
+    Given :math:`\lambda =`'length_scale' and :math:`\rho =`'output_scale', the squared
+    exponential kernel is defined as
+    :math:`k: \mathbb{R}^d\times \mathbb{R}^d \to \mathbb{R}`,
+    :math:`k(x, y) = \rho \exp(\frac{||x-y||^2}{2 \lambda^2})` where
+    :math:`||\cdot||` is the usual :math:`L_2`-norm.
+
     :param length_scale: Kernel smoothing/bandwidth parameter
```

Comment:
Can we add the symbols to these param docstring lines now please. E.G. ":param length_scale: Kernel smoothing/bandwidth parameter, :math:`\lambda`"